uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/manager-leave/components/manager-leave-requests.js

```diff
@@ -22,8 +22,8 @@ define([
 
     var vm = Object.create(this),
       actionMatrix = {
-        'waiting_approval': ['respond', 'cancel'],
-        'more_information_requested': ['edit', 'cancel'],
+        'awaiting_approval': ['respond', 'cancel'],
```

Comment:
Move all the status names to `shared-settings` and use the same in each file.

If backend values are changed, we should change the values only at a single place.